momento-sdk/src/main/java/momento/sdk/ValidationUtils.java

```diff
@@ -4,6 +4,7 @@
 
 // Should rely on server for all validations. However, there are some that cannot be delegated and
 // instead fail in grpc client, like providing null inputs or a negative ttl.
+// TODO: validation should not let exceptions escape from cache methods
```

Comment:
This is here because validation will currently throw exceptions from the new methods even though that is incorrect. I didn't want to fix it until I had all the responses switched over. That pr will immediately follow the last response conversion pr.

Reply:
this is fine for now, but you should also discuss this with @pgautier404 and @malandis before you do the work to change the behavior.  For some types of exceptions that are just straight up user bugs, we do actually throw exceptions.  I think Schwern wrotes some notes about it in one of our standards-and-practices docs.